conf.py

```diff
@@ -18,19 +18,19 @@
 
 
 # -- Aplus configuration --------------------------------------------------
-course_open_date = '2016-06-01'
-course_close_date = '2020-06-06'
+course_open_date = '2020-04-21'
+course_close_date = '2021-05-01'
```

Comment:
Let's set the `course_close_date` further into the future. Year 2030.

Reply:
I think this is a good reminder to update the course content.